crates/hyperqueue/src/client/commands/worker.rs

```diff
@@ -140,24 +166,58 @@ pub async fn start_hq_worker(
     Ok(())
 }
 
+fn lost_worker_info_to_str(worker: &WorkerInfo) -> String {
+    let worker_process = get_worker_ended_reason(worker);
+    if worker_process.is_none() {
+        String::from("running")
+    } else {
+        worker_process.as_ref().unwrap().to_string()
+    }
+}
+
+fn get_worker_ended_reason(worker: &WorkerInfo) -> Option<LostWorkerReasonInfo> {
+    if worker.ended.is_some() {
+        Some(worker.ended.as_ref().unwrap().reason.clone())
+    } else {
+        None
+    }
+}
+
 pub async fn get_worker_list(
     connection: &mut ClientConnection,
     include_offline: bool,
```

Comment:
`include_offline` is no longer needed, this function should just take `connection` and `filters: Set<WorkerFilter>`.